test/Lachain.CoreTest/IntegrationTests/BlocksTest.cs

```diff
@@ -223,6 +229,58 @@ public void Test_Block_With_Txs_Execution()
             }
             ExecuteTxesInSeveralBlocks(randomReceipts);
         }
+        
+        [Test]
+        public void Test_Test()
```

Comment:
I think it is better to give this test more explanatory name.)

Reply:
Sorry, forgot to update